apko.yaml

```diff
@@ -1,7 +1,8 @@
 package:
   name: apko
+  # When bumping the version check if the CVE/GHSA mitigations below can be removed.
```

Comment:
Experience in Alpine shows me that comments like this never really get read.  Maybe we need something other than a comment for these cases, like gating the patching with an `if` expression.

Reply:
https://github.com/chainguard-dev/melange/issues/384

Reply:
FWIW I sent this today which might help here: https://github.com/chainguard-dev/melange/commit/ece24c2854a2956bf48c539b312cde654765e6d5#diff-e9720019c7b001a10aa87ca89ef44a50a3942cfb22ea8c16ea3dd009d9992a0c

Reply:
I think Dan's update to `go/build` helps us move this idea forward. I think what's left is to ensure that the `go get` is applied only if the specified module version is _later_ than what you'd see with `go list -m all`.

IMHO, that's the point where I think it makes sense to tie in automation that removes the version bumps. I think that functionality would probably go into `wolfictl update`.

Down the road we could also have dependabot-like automation (e.g. in `wolfictl`) that can automatically _add_ those version bump snippets — and add corresponding "fix" advisory entries — if we can determine that such a bump would resolve the vulnerability.